q2_moshpit/plugin_setup.py

```diff
@@ -560,3 +562,48 @@
                 "visualizations summarizing the results.",
     citations=[citations["manni_busco_2021"]],
 )
+
+plugin.methods.register_function(
+    function=q2_moshpit.prodigal.predict_genes_prodigal,
+    inputs={
+        'mags': FeatureData[MAG]
+    },
+    input_descriptions={
+        'mags': 'MAGs for which one wishes to predict genes.'
+    },
+    parameters={
+        "translation_table_number": Int % Range(1, 26)
```

Comment:
```suggestion
        "translation_table_number": Str % Choices(['1', '2', '3', '4', '5', '6', '9',
                                                   '10', '11', '12', '13', '14', '15',
                                                   '16', '21', '22', '23', '24', '25'])
```
It really bugs me that you can input a parameter that leads prodigal to fail.
With this change, only the working parameters can be inputted. If a wrong one is given, there is directly an error message, and you don't have to open the log to see what the problem is.
Unfortunately, it's not possible to use `Choices` with integers, but it does not change anything for the user if the parameters are strings.